backend/Entities/Utils/ParseSqlQuery.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Entities.Utils
```

Comment:
Are you sure that Utils is C# class?

Reply:
what the problem?
There will be no collision or smth alike

Reply:
I created Utils folder in Entities part of project. This folder contains static class that have methods (defined for my specific case) for parsing sql-query result. What is wrong?